<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>720</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_plot">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>720</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Plot1</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>720</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Plot2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
